cypress/e2e/marc/marc-bibliographic/edit-marc-bib/edit-imported-marc-bib-record-file-without-required-number-008-position.cy.js

```diff
@@ -0,0 +1,95 @@
+import getRandomPostfix from '../../../../support/utils/stringTools';
+import { RECORD_STATUSES } from '../../../../support/constants';
+import Permissions from '../../../../support/dictionary/permissions';
+import TopMenu from '../../../../support/fragments/topMenu';
+import DataImport from '../../../../support/fragments/data_import/dataImport';
+import Users from '../../../../support/fragments/users/users';
+import JobProfiles from '../../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../../support/fragments/data_import/logs/logs';
+import MarcAuthority from '../../../../support/fragments/marcAuthority/marcAuthority';
+import QuickMarcEditor from '../../../../support/fragments/quickMarcEditor';
+import InventoryInstance from '../../../../support/fragments/inventory/inventoryInstance';
+import TopMenuNavigation from '../../../../support/fragments/topMenuNavigation';
+
+describe('MARC -> MARC Bibliographic -> Edit MARC bib', () => {
+  const testData = {
+    createdRecordIDs: [],
+  };
+
+  const marcFile = {
+    marc: 'marcBibFileForC387455.mrc',
+    fileName: `testMarcFileC387455.${getRandomPostfix()}.mrc`,
+    jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+    numOfRecords: 1,
+  };
+
+  const firstInstanceTitle = 'Extra 008 positions Vanity fair.';
+  const secondInstanceTitle = 'Too few 008 positions Vanity fair.';
+
+  before('Create user and data', () => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorAll.gui,
+      Permissions.moduleDataImportEnabled.gui,
+    ]).then((createdUserProperties) => {
+      testData.userProperties = createdUserProperties;
+
+      cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(() => {
+        DataImport.verifyUploadState();
+        DataImport.uploadFileAndRetry(marcFile.marc, marcFile.fileName);
+        JobProfiles.waitFileIsUploaded();
+        JobProfiles.search(marcFile.jobProfileToRun);
+        JobProfiles.runImportFile();
+        JobProfiles.waitFileIsImported(marcFile.fileName);
+        Logs.checkStatusOfJobProfile('Completed');
+      });
+
+      cy.login(testData.userProperties.username, testData.userProperties.password, {
+        path: TopMenu.dataImportPath,
+        waiter: DataImport.waitLoading,
+      });
+    });
+  });
+
+  after('Delete test data', () => {
+    if (testData.createdAuthorityID) MarcAuthority.deleteViaAPI(testData.createdAuthorityID);
```

Comment:
Please replace this line with step to delete both imported instance records. No authorities were imported here
You can use this method: `InventoryInstance.deleteInstanceViaApi(<<id>>)`